spec/factories/custom_field_factory.rb

```diff
@@ -288,134 +183,29 @@
         evaluator.projects.each do |project|
           project.work_package_custom_fields << custom_field
         end
-
-        if evaluator.projects.any? || evaluator.types.any?
-          # As the request store keeps track of the available custom fields of work packages
-          RequestStore.clear!
-        end
```

Comment:
Is this deleted because there's already an `after(:create)` hook above that clears the `RequestStore` anyway?

Reply:
Yep, exactly.